PUBLISH.md

```diff
@@ -1,33 +1,56 @@
-# Releasing a new version
+# Version management
 
-To be able to do `docker push` you need to have your Docker Hub account added to the dedis organisation.
-Here are the steps we use to make a conode binary release:
-```
-git checkout master
-git diff -> be sure you have no outstanding changes
-git log
-```
-review to be sure this is where the tag should be
+For the latest version of the conodes, please use 
+https://github.com/c4dt/byzcoin, which provides a continuous release of valid
+ conodes.
+
+For the `@dedis/kyber` and `@dedis/cothority` npm libraries, we provide a
+ development and a somewhat stable build.
+The development build is always up-to-date with the latest commit and can be
+ used with a reference to `@dedis/cothority@dev`:
+```bash
+npm i -s @dedis/cothority@dev
+```
+This pins the library to the latest available development version.
+
+## Releasing a new version for npm @dedis/*
+
+The npm-packages should follow semantic versioning.
```

Comment:
Could provide a link to https://semver.org/, in case people are unfamiliar with it.